src/plugins/intel_cpu/src/emitters/snippets/utils/utils.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "utils.hpp"
+
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+
+#include "emitters/utils.hpp"
+#include "openvino/core/except.hpp"
+#include "openvino/core/type.hpp"
+#include "snippets/lowered/expression_port.hpp"
+#include "snippets/lowered/expressions/buffer_expression.hpp"
+#include "snippets/op/loop.hpp"
+#include "snippets/op/memory_access.hpp"
+#include "snippets/utils/utils.hpp"
+#include "utils/general_utils.h"
+
+namespace ov::intel_cpu::emitters::snippets::utils {
```

Comment:
Can we leave these helpers in `ov::intel_cpu::utils` namespace for now? I understand your idea, but I think we need to do such changes as new namespaces introduction for all the affected code (e.g. should we place all the emitter classes under  `emitters` namespace)?
Let's discuss this question offline and apply the change (if needed) in a separate PR